src/scss/initial/_links.scss

```diff
@@ -68,3 +68,21 @@ a {
     text-decoration: none;
   }
 }
+
+// For links styled as buttons, underlines
+// are removed and colors match styles applied
+// buttons
+[data-btn~='link-button'] {
+  --link: var(--bg);
+  --underline-thickness: 0px;
+  --underline-thickness--default: 0px;
+  --underline-color--default: transparent;
+  --underline-color: transparent;
+  --link-focus: var(--bg);
+
+  &:hover,
+  &:focus {
+    --underline-thickness: 0px;
+    --underline-color: transparent;
```

Comment:
```suggestion
    --underline-color: transparent;
    --underline-thickness: 0;
```

Reply:
fixed in https://github.com/oddbird/oddleventy/pull/808/commits/15b0a3f91ab66ebdb1aaf83febbce025309cdbfe